src/components/Header/index.tsx

```diff
@@ -111,8 +124,9 @@ const GordonHeader = ({ onDrawerToggle }) => {
           className={styles.tab}
           icon={icon}
           label={name}
-          component={ForwardNavLink}
-          to={route}
+          component={forwardRef<any, Omit<RouterLinkProps, 'to'>>((props, ref) => (
+            <NavLink ref={ref} to={route} {...props} />
+          ))}
```

Comment:
I think you can just use plain old `NavLink` here, although I'm not sure:
```suggestion
          component={NavLink}
          to={route}
```